plot_benchmark.py

```diff
@@ -0,0 +1,156 @@
+import os
```

Comment:
This file and `benchmark.ipynb` should be moved into an `examples/benchmarks` directory. And the current contents of the `examples` directory should be moved into an `examples/tutorials` directory